broadcast/src/main/java/org/elasticsoftware/elasticactors/broadcast/Broadcaster.java

```diff
@@ -139,21 +140,36 @@ public void handleAdd(Add add,BroadcasterState state) throws Exception {
     }
 
     @MessageHandler
-    public void handleUpdateThrottleConfig(UpdateThrottleConfig updateThrottleConfig, BroadcasterState state) {
```

Comment:
same comment about backwards compatibility

Reply:
We're shifting how throttling is done here, from a potentially problematic solution in which one actor could interfere with another, to a more encapsulated one. I don't know how we can expect to retain backwards compatibility with anything here. The point is to have the message not be usable. 

The only reason I didn't remove it entirely is because of possible messages that are on the broker.

Reply:
ok